Sources/SmithyTestUtil/XCTAssertions.swift

```diff
@@ -46,3 +46,20 @@ public func XCTAssertXMLDataEqual(
         XCTFail("Failed to evaluate XML with error: \(error)", file: file, line: line)
     }
 }
+
+public func XCTAssertFormURLDataEqual(
```

Comment:
Helper function that uses the new Form URL comparator to assert that two FormURL bodies are equivalent.